internal/e2e/e2e_index_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package build
+package e2e
```

Comment:
I moved (almost all) of these tests to the `e2e` package to avoid circular dependencies.